nephio-ansible-install/NIND.md

```diff
@@ -0,0 +1,55 @@
+# nephio-in-docker (nind)
+
+## tldr
+
+nephio-in-docker (nind) packages the nephio-ansible-install into a [Docker](https://docs.docker.com/get-docker/) container following the setup used in the [nephio ONE summit 2022 workshop](https://github.com/nephio-project/one-summit-22-workshop).
+In addition nind also runs a [VS Code OSS](https://github.com/microsoft/vscode) workstation instance providing a browser based development and test enviroment.
+
+To build and run nephio-in-docker execute the commands listed below:
+
+```bash
+git clone https://github.com/nephio-project/one-summit-22-workshop.git
+cd one-summit-22-workshop/nephio-ansible-install
+docker build -t nind .
+docker run --name=nind --rm --env='DOCKER_OPTS=' --volume=/var/lib/docker --privileged \
+  --cgroup-parent=nephio.slice --restart=no -d -p 8080:80 -p 7007:7007 -p 3000:3000 nind
+```
+Note that it will take about 10 mins for nephio to set up after starting the container as it executes the commands in the [installation script](/nephio-ansible-install/nind/150_install-nephio.sh).
+You can follow the nephio installation progress by connecting to the nind container and access the installation log files.
+
+```bash
+# connect to the nind container 
+docker exec -it nind bash
+
+# change to the nephio installation directory
+cd /nephio-installation
+
+# you will see 5 log files coming up logging the progress of the installation steps.
+# 00_prereq.out, 01_gitea.out, 02_repos.out, 03_clusters.out and 04_nephio.out
+# when the last step starts executing follow the progress by tailing its log file.
+tail -f 04_nephio.out
+
+# once the installtion completes you will see a line like the one below.
+PLAY RECAP *********************************************************************
+localhost                  : ok=22   changed=18   unreachable=0    failed=0    skipped=9    rescued=0    ignored=0
+```
+
+You can build and run nind on any laptop or workstaton that packs-the-punch to let nephio fly.
```

Comment:
For the workshop VM, we recommended a VM with 8 vcpus, 32 GB RAM and 50 GB disk space. I am guessing that the NIND container is to be run on a compute machine with above requirements, right?

Reply:
it would be the same. I tested with a 4vcpu/16G as well, so this worked for me. now the more workloads we add not sure if this will still hold or not.

Reply:
so lets stay with the 32G, 8vcpu and we also have to specify the disk of 64G or so

Reply:
The next sentence in the NIND.md file actually refers back to the main README.md with the workshop VM spec.
As @henderiw mention nind also runs with 4vcpu/16G (which i most often used during the develpment).
However, would also advocate to leave head room as the resource demands will only grow as the implementation progresses.